omod/src/main/webapp/template/headerFull.jsp

```diff
@@ -175,7 +175,7 @@
 			</openmrs:forEachAlert>
 
 			<c:if test="${msg != null}">
-				<div id="openmrs_msg"><openmrs:message code="${msg}" text="${msg}" arguments="${msgArgs}"  htmlEscape="false" /></div>
+				<div id="openmrs_msg"><openmrs:message code="<c:out value='${msg}'/>" text="<c:out value='${msg}'/>" arguments="<c:out value='${msgArgs}'/>" htmlEscape="false" /></div>
```

Comment:
@jmmacdo4 is working on this as part of EMPT54 in https://github.com/openmrs/openmrs-module-legacyui/pull/136

If we get to a general solution there, we might not need this PR.